scripts/qtsupport.py

```diff
@@ -28,6 +29,138 @@
 from jmbitcoin.bip21 import decode_bip21_uri
 from jmclient import (jm_single, validate_address, get_tumble_schedule)
 
+class MakerDialog(QDialog):
+
+    parameter_names = ['CJ Offer type', 'CJ Absolute fee (sats)',
+              'CJ Relative fee (decimal)',
+              'Tx fee contribution (sats)',
+              'Minimum CJ size (sats)']
+    parameter_tooltips = [
+     "Type of offer; absolute is a fixed price in sats,\n"
+     "relative is a decimal fraction of coinjoin amount.",
+     "Satoshis fee (integer) if absolute offer type chosen.",
+     "Decimal fraction of coinjoin amount offered as fee\n"
+     "if relative offer type chosen.",
+     "Satoshis contribution (integer) to bitcoin transaction fee.",
+     "Minimum size in satoshis of coinjoin allowed."]
+    parameter_types = [str, int, float, int, int]
+    parameter_settings = ['Relative fee', 500, 0.0002, 100, 1000000]
```

Comment:
"Minimum CJ size (sats)" should be changed to 100000, which is default for cli since #570.

Reply:
But in context of #727, should be read from the config in future, not being hardcoded here.